specutils/spectra/spectrum1d.py

```diff
@@ -91,17 +92,54 @@ def to_flux(self, unit):
             The converted flux array.
         """
         new_data = self.flux.to(
-            unit, equivalencies=eq.spectral_density(self.dispersion))
+            unit, equivalencies=eq.spectral_density(self.spectral_axis))
 
         self._data = new_data.value
         self._unit = new_data.unit
 
         return self.flux
 
-    def to_dispersion(self, unit, rest, type="doppler_relativistic"):
+    @property
+    def frequency(self):
+        return self._spectral_axis.to(u.GHz, u.spectral())
+
+    @property
+    def wavelength(self):
+        return self._spectral_axis.to(u.AA, u.spectral())
+
+    @property
+    def energy(self):
+        return self._spectral_axis.to(u.ev, u.spectral())
+
+    @property
+    def velocity_convention(self):
+        return self._velocity_convention
+
+    @velocity_convention.setter
+    def velocity_convention(self, value):
+        if value not in ('relativistic', 'optical', 'radio'):
+            raise ValueError("The allowed velocity conveintions are 'optical' "
+                             "(linear with respect to wavelength), 'radio' "
+                             "(linear with respect to frequency), and 'doppler'.")
```

Comment:
Should this error message include ```'relativistic'```?  Also, should the if statement check also allow ```'doppler'``` based on the error message?